docs/user/explanations/flyscanning.rst

```diff
@@ -0,0 +1,63 @@
+Flyscanning
```

Comment:
From discussion: this should go in the cookbook rather than ophyd-async

Reply:
I think for the sake of progress this goes into ophyd-async now, then I pull put on the cook-book issue to move it when it becomes relevant. It's more useful existing in the wrong place than not existing in either place.